types/evidence.go

```diff
@@ -220,8 +225,9 @@ func (l *LightClientAttackEvidence) ABCI() []abci.Evidence {
 	abciEv := make([]abci.Evidence, len(l.ByzantineValidators))
 	for idx, val := range l.ByzantineValidators {
 		pb := abci.Validator{
-			Address: val.Address,
-			Power:   val.StakingPower,
+			Address:     val.Address,
```

Comment:
How about update to `Validator:        TM2PB.Validator(val),`?

Reply:
Sounds good, fix it